Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -80,79 +82,59 @@ void AudioPane::CreateWidgets()
-  m_dolby_pro_logic = new QCheckBox(tr("Dolby Pro Logic II Decoder"));
-
-  if (m_latency_control_supported)
-  {
-    m_latency_label = new QLabel(tr("Latency:"));
-    m_latency_spin = new QSpinBox();
-    m_latency_spin->setMinimum(0);
-    m_latency_spin->setMaximum(200);
-    m_latency_spin->setToolTip(
-        tr("Sets the latency in milliseconds. Higher values may reduce audio "
-           "crackling. Certain backends only."));
-  }
-
-  m_dolby_pro_logic->setToolTip(
-      tr("Enables Dolby Pro Logic II emulation using 5.1 surround. Certain backends only."));
-
-  auto* dolby_quality_layout = new QHBoxLayout;
-
+  m_backend_combo =
+      new ConfigStringChoice(AudioCommon::GetSoundBackends(), Config::MAIN_AUDIO_BACKEND);
+  m_dolby_pro_logic = new ConfigBool(tr("Dolby Pro Logic II Decoder"), Config::MAIN_DPL2_DECODER);
   m_dolby_quality_label = new QLabel(tr("Decoding Quality:"));
 
-  m_dolby_quality_slider = new QSlider(Qt::Horizontal);
-  m_dolby_quality_slider->setMinimum(0);
-  m_dolby_quality_slider->setMaximum(3);
-  m_dolby_quality_slider->setPageStep(1);
-  m_dolby_quality_slider->setTickPosition(QSlider::TicksBelow);
-  m_dolby_quality_slider->setToolTip(
-      tr("Quality of the DPLII decoder. Audio latency increases with quality."));
-  m_dolby_quality_slider->setTracking(true);
-
-  m_dolby_quality_low_label = new QLabel(GetDPL2QualityLabel(AudioCommon::DPL2Quality::Lowest));
-  m_dolby_quality_highest_label =
-      new QLabel(GetDPL2QualityLabel(AudioCommon::DPL2Quality::Highest));
-  m_dolby_quality_latency_label =
-      new QLabel(GetDPL2ApproximateLatencyLabel(AudioCommon::DPL2Quality::Highest));
-
-  dolby_quality_layout->addWidget(m_dolby_quality_low_label);
-  dolby_quality_layout->addWidget(m_dolby_quality_slider);
-  dolby_quality_layout->addWidget(m_dolby_quality_highest_label);
+  QStringList quality_options{
+      tr("Lowest Quality (528), Fastest (~10 ms)"), tr("Low Qualiity (1024), Fast (~20 ms)"),
+      tr("High Quality (2048), Slow (~40 ms)"), tr("Highest Quality (4096). Slowest (~80 ms)")};
+
+  m_dolby_quality_combo = new ConfigChoice(quality_options, Config::MAIN_DPL2_QUALITY);
 
   backend_layout->setFormAlignment(Qt::AlignLeft | Qt::AlignTop);
   backend_layout->setFieldGrowthPolicy(QFormLayout::AllNonFixedFieldsGrow);
   backend_layout->addRow(m_backend_label, m_backend_combo);
-  if (m_latency_control_supported)
-    backend_layout->addRow(m_latency_label, m_latency_spin);
 
 #ifdef _WIN32
+  std::vector<std::pair<QString, QString>> wasapi_options;
+  wasapi_options.push_back(
+      std::pair<QString, QString>{tr("Default Device"), QStringLiteral("default")});
+
+  for (auto string : WASAPIStream::GetAvailableDevices())
+  {
+    wasapi_options.push_back(std::pair<QString, QString>{QString::fromStdString(string),
+                                                         QString::fromStdString(string)});
+  }
+
   m_wasapi_device_label = new QLabel(tr("Device:"));
```

Comment:
`Output Device:`

(since you're already refactoring the panel, I'll ask for this too)